platform/admin/core/admin-core-migration-commands/pom.xml

```diff
@@ -143,17 +143,17 @@
                     <limit implementation="org.codice.jacoco.LenientLimit">
                       <counter>INSTRUCTION</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.98</minimum>
+                      <minimum>0.85</minimum>
                     </limit>
                     <limit implementation="org.codice.jacoco.LenientLimit">
                       <counter>BRANCH</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>0.99</minimum>
+                      <minimum>0.70</minimum>
```

Comment:
Would it work to use `ConsoleOutput`?

then test console output with `assertThat(consoleOutput.getOutput(), containsString(str));`

Reply:
I have to mock the response otherwise I get an NPE when I try to see what the response is: 
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveAllCommandTest.java#L230
I think I could take this same approach.

Reply:
Now that I'm looking at this again, since we're already mocking session this should be relatively trivial to add in to my test in the same way. I'll update the coverage.